packages/v4/src/content/design-guidelines/usage-and-behavior/modal/modal.md

```diff
@@ -5,10 +5,10 @@ hideTOC: true
 showTitle: true
 ---
 
-Use modal dialogs load and display important information without navigating away from the current page.
+Use modal dialogs to load and display important information without navigating away from the current page.
 
 ## Primary elements
-![Modal](./img/modal.png)
+![Modal](<img src="./img/modal.png" alt="Modal" width="756"/>)
```

Comment:
The syntax for this should be:
`<img src="./img/modal.png" alt="Modal" width="756"/>`

I'll comment on the following ones as well!